Add mouse camera jump fix.

phamd opened the pull request:
Offsets and bytes taken from https://github.com/brndd/vanilla-tweaks/issues/17
For reference, I used: Vanilla.txt, Turtle WoW Camera Patch.exe, and Vanilla Camera Patch.exe.
Tested on Turtle WoW.
I can make further changes if needed.

phamd committed:
Add mouse camera jump fix.

brndd commented:
Thanks for this!

I saw this fix when the issue was made, but haven't had the time to test it out yet -- and I'm not sure if I could easily do that anyway, because I don't think I've noticed the cursor jumping problem very often on Wine. In any case, I'll give your patch a try soon.

Have you played with this, and if so, have you noticed any side effects?

phamd commented:
Without the patch, every 3 right clicks would send my mouse to the center of the screen (with Hardware Cursor enabled). So the fix is definitely working and I haven't encountered any side effects yet.

I also just set up VanillaFixes + nampower and it feels good so far.

brndd committed:
Minor reformatting.

Change the name of the camera glitch fix from "mouse camera jump fix"
to "camera rotation skip glitch fix"

Since this patch is quite large, I decided to format the longer hex values
so that they align with a hex editor view of the client, to make it easier
to manually compare them to the executable should the need arise.
